code/goonstation/modules/power/power_transmission_laser/transmission_laser.dm

```diff
@@ -0,0 +1,507 @@
+#define MINIMUM_POWER 1 MW
+
+/// How much energy do you have to sell in order to get an announcement. Increases each time we reach it.
+GLOBAL_VAR_INIT(announcement_threshold, 1 MJ)
```

Comment:
Maybe this could be a static variable on the PTL instead of a global var?

Reply:
Sounds good. Done